cli/src/commands/dsl/validate.ts

```diff
@@ -44,11 +96,14 @@ export default class Validate extends BaseCommand {
       }
     );
 
-    const program = ts.createProgram(files, DEFAULT_COMPILER_OPTIONS);
+    const TSConfig = (await this.getTSConfig()) || DEFAULT_COMPILER_OPTIONS;
```

Comment:
Should that be a `??`  since we only want to fall back to the default if `undefined` is returned, not a falsey value.